flink-runtime/src/test/java/org/apache/flink/runtime/minicluster/MiniClusterConfigurationTest.java

```diff
@@ -74,4 +78,15 @@ public void testSchedulerType_notOverriddenIfExistingInConfig() {
 			JobManagerOptions.SCHEDULER.defaultValue(),
 			miniClusterConfiguration.getConfiguration().getString(JobManagerOptions.SCHEDULER));
 	}
+
+	@Test
+	public void testDefaultTaskExecutorMemoryConfiguration() {
+		final MiniClusterConfiguration miniClusterConfiguration = new MiniClusterConfiguration.Builder().build();
+		final Configuration actualConfiguration = miniClusterConfiguration.getConfiguration();
+		final Configuration expectedConfiguration = MiniClusterConfiguration.adjustTaskManagerMemoryConfigurations(new Configuration());
+
+		assertThat(actualConfiguration.get(TaskManagerOptions.SHUFFLE_MEMORY_MIN), is(expectedConfiguration.get(TaskManagerOptions.SHUFFLE_MEMORY_MIN)));
+		assertThat(actualConfiguration.get(TaskManagerOptions.SHUFFLE_MEMORY_MAX), is(expectedConfiguration.get(TaskManagerOptions.SHUFFLE_MEMORY_MAX)));
+		assertThat(actualConfiguration.get(TaskManagerOptions.MANAGED_MEMORY_SIZE), is(expectedConfiguration.get(TaskManagerOptions.MANAGED_MEMORY_SIZE)));
```

Comment:
I would change the assertions to test the actual values because now the test would also pass if `adjustTaskManagerMemoryConfigurations` did not do anything.